src/components/SettingsButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+class SettingsButton extends React.Component {
+  render() {
+    return (
+      <div>
+        <Link to="./settings">
+          <button data-testid="btn-settings">Settings</button>
```

Comment:
Sempre que usar uma tag 'button', insira o tipo dele (type) :-)
```suggestion
          <button data-testid="btn-settings" type="button">Settings</button>
```